packages/core/src/types/other.ts

```diff
@@ -10,9 +10,7 @@ export interface DeploymentResult {
   extraData: any;
 } 
 
-export interface PipObject {
-  [key: string]: any;
-}
+export type PipObject = Record<string, any>
```

Comment:
Good improvements, just figure out another "good first issue": use TypeScript `Record` instead of custom object.

/cc @utkobe

Reply:
Added: https://github.com/alibaba/pipcook/issues/142